README.md

```diff
@@ -92,12 +102,12 @@ require'nvim-web-devicons'.setup {
 
 ### Get Icon
 
-Get the icon for a given file by passing in the `name`, the `extension` and an _optional_ options `table`.
+Get the icon for a given file by passing in the `name`, the `pre_or_ext` and an _optional_ options `table`.
 The name is passed in to check for an exact match e.g. `.bashrc` if there is no exact name match the extension
 is used. Calls `.setup()` if it hasn't already ran.
 
 ```lua
-require'nvim-web-devicons'.get_icon(filename, extension, options)
+require'nvim-web-devicons'.get_icon(filename, pre_or_ext, options)
```

Comment:
This will break existing behaviour - a prefixed icon will be returned where none was before.